### PR TITLE
Requires protobuf.

### PR DESCRIPTION
**Pull request by DaviesX**

### CODE REVIEW

**Comment by zhaonian** [comment]
why is protobuf needed? :/

File: build.gradle [hunk 1]
```diff
@@ -6,13 +6,15 @@ buildscript {
     repositories {
         google()
         gradlePluginPortal()
+        mavenCentral()
```
Comment (jaredsburrows) [inline build.gradle]: `gradlePluginPortal()` pulls from both `jenter()` and `mavencentral()`.

Comment (DaviesX) [reply build.gradle]: removed

**Comment by DaviesX** [comment]
> why is protobuf needed? :/

I need to place a configuration file in each image repository in the form a text proto. Text proto is easier to work with than other config file formats (e.g. XML/JSON) in my opinion.

**Comment by zhaonian** [comment]
ok. it looks like you added the plugin for it to generate java code, but did not add the actual protobuf dependency?
maybe `implementation 'com.google.protobuf:protobuf-lite:3.0.0'`?

if you just want a format tool for textproto, probably just use the plugin for android studio, maybe not needed to add anything here.

**Review by zhaonian — APPROVED** [review]

**Pull request merged**